tests/integration/test_ethernet.py

```diff
@@ -0,0 +1,93 @@
+import os
```

Comment:
This file is missing a copyright header, it should look like:

```
# -*- coding: utf-8 -*-
# SPDX-License-Identifier: BSD-3-Clause
```

Reply:
I added it, thanks!